### ISSUE TITLE
Write a language tutorial

### ISSUE THREAD

**Issue by apaszke** [google-research/dex-lang#425]
We've had a language tutorial a long time ago, but it's not longer accurate so we had to remove it. We have no way of onboarding more people, short of meeting with them in person, which doesn't scale great.

**Comment by srush** [comment]
Can I try this? Is it okay if it is somewhat casual in its tone?

**Comment by dougalm** [comment]
Of course, that would be amazing! Feel free to make it as casual or as formal as you'd like.

**Comment by apaszke** [comment]
I would even say that casual might even be preferred for an introductory tutorial!
